src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,101 @@
+package mate.jdbc.dao;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class ManufacturerDaoImpl implements ManufacturerDao {
+    private static final String GET_ALL = "SELECT * FROM manufacturers WHERE is_deleted = FALSE;";
+    private static final String GET_BY_ID
+            = "SELECT * FROM manufacturers WHERE is_deleted = FALSE AND id = ?;";
+    private static final String UPDATE = "UPDATE manufacturers SET name = ?,"
+            + " country = ? WHERE is_deleted = FALSE AND id = ?;";
+    private static final String DELETE_BY_ID
+            = "UPDATE manufacturers SET is_deleted = true WHERE id = ?;";
```

Comment:
```suggestion
    private static final String DELETE_BY_ID_QUERY
            = "UPDATE manufacturers SET is_deleted = TRUE WHERE id = ?;";
```
Let's add QUERY suffix to your constants here so that their purpose is more clear